Python/RestNotifier/Reademe.MD

```diff
@@ -0,0 +1,42 @@
+## Rest Notifier in Python
+
+### What is Rest Notifier ?
+``` It is a simple python script which prompts you to take break after certain period of time so as to relax yourself
+Here you can give the amount of time after you want the break and no of breaks you want.
+Moreover you can even change the song you like.
+```
+
+
+### Introduction
+```
+A lot of us suffer from backpain, eye-strain, and other related problems while working for longer hours without taking any breaks. So I spent some minutes writing a Python script that reminds us to take a break after every 20 minutes for 5 minutes while we are working. As a reminder, it plays your favorite song.
+
+The module i have used are webbrowser and Time.
+```
+
+
+
+### Code
+
+### Step 1: Import Required Module
+```
+import webbrowser
+import time
+```
+### Step 2: Run the code :
+```
+cd RestNotifier
+python notifier.py
+```
+### Output
+```When you run the file you will be asked about number of break and the time in second after you want the breaks
+Then after specific amount of time you will be redirected to a relaxation song.

```

Comment:
this can be optional. So provide an option to choose if user wants to listen to music ?